### ISSUE TITLE
[libretro] segfaulting when loading a game

### ISSUE THREAD

**Issue by BPaden** [inolen/redream#124]
```
Thread 1 "retroarch" received signal SIGSEGV, Segmentation fault.
0x00007f20b263323d in mp_begin_frame (mp=0x0, width=640, height=480) at /home/aaron/Documents/progs/redream/src/render/microprofile.cc:254
254	  mp->viewport_width = width;
```

**Comment by inolen** [comment]
Should be fixed with 1b85abdfabc3ecf54353cefa4ad7f99c11329b7c.

**Comment by BPaden** [comment]
That it do. :)
